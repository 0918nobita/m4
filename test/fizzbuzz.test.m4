include(fizzbuzz.m4)dnl
fizzbuzz(1, 30)
