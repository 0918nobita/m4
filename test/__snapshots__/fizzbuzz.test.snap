1
2
3 <- Fizz
4
5 <- Buzz
6 <- Fizz
7
8
9 <- Fizz
10 <- Buzz
11
12 <- Fizz
13
14
15 <- FizzBuzz
16
17
18 <- Fizz
19
20 <- Buzz
21 <- Fizz
22
23
24 <- Fizz
25 <- Buzz
26
27 <- Fizz
28
29
30 <- FizzBuzz
